test/integration/core/timers.tap.js

```diff
@@ -185,66 +185,99 @@ tap.test('nextTick with extra args', function testNextTick(t) {
   }
 })
 
-tap.test('clearTimeout', function testNextTick(t) {
```

Comment:
There' was a duplicate test in here I cleared out. That makes this diff look weird unfortunately.